sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/StorageHelper.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter
+{
+    internal static class StorageHelper
+    {
+        private static string defaultStorageDirectory;
+        private const string nonWindowsVarTmp = "/var/tmp/";
```

Comment:
Could we discuss and document the behavior across different language SDKs to make sure there is consistency?